Xarray/xarray-oceanhackweek21.ipynb

```diff
@@ -28,7 +28,7 @@
   },
```

Comment:
<p>Its just this cell and the one below that are new.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oceanhackweek/ohw-tutorials/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>